ark/spLDA/processing.py

```diff
@@ -0,0 +1,86 @@
+import functools
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import os
+import pickle
+from ark.utils.spatial_lda_utils import check_format_cell_table_args
+# from spatial_lda.featurization import *
+# from spatial_lda.visualization import plot_adjacency_graph()
+
+
+def format_cell_table(cell_table, markers=None, clusters=None, fovs="all_fovs"):
+    """
+    Formats a cell table containing one for more fields of view to be compatible with the spatial_lda library.
+    Args:
+        cell_table (pd.DataFrame):
```

Comment:
When listing datatypes in the docstring, you'll need to use the full class name so it can get mocked up properly. Here, it should be `pandas.DataFrame`.